### PR TITLE
Add JSOM and iD Editor Quickstarts (for Open Street Map)

### PR DESCRIPTION
**Pull request by camerons**
These quickstarts are derived from learnOSM which are licenced CC0 which fits with osgeo live licences.
Published Source: http://learnosm.org/en/josm/start-josm/
http://learnosm.org/en/beginner/id-editor/

Source in Git: https://github.com/hotosm/learnosm/blob/gh-pages/_posts/en/1900-12-21-start-josm.md
https://github.com/hotosm/learnosm/blob/gh-pages/_posts/en/0200-12-23-id-editor.md

### CODE REVIEW

File: en/quickstart/josm_quickstart.rst [hunk 1]
```diff
@@ -234,9 +234,8 @@ corner of that window. Alternatively you can choose delete from the
 context menu of the layer. In order to continue editing you first need
 to download real data. This will be explained in the next chapter.
 
-Summary
--------
+What Next?
+----------
+
+- Try the LeanOSM documentation, from which this quickstart was derived: http://learnosm.org/en/josm/start-josm/
```
Comment (althio) [inline en/quickstart/josm_quickstart.rst]: LearnOSM

**Comment by althio** [comment]
Is the file 'sample.osm' included somewhere?
It is referenced in the JOSM page for sandbox-editing.

**Comment by camerons** [comment]
@althio well noticed re needing the 'sample.osm' dataset. From what I can gather it is not included.
Looking at OSGeo-Live11.0beta1 I see we have the following files installed in ~/data/osm:
Boston_MA.osm.bz2
feature_city.osm.bz2

To date, someone has been updating the OSM dataset to point to the FOSS4G city, which is fine, however it means that any quickstarts showing OSM need to be updated as well. I believe we need a static OSM dataset for this. I suspect the sample.osm would be a good candidate?

Incorporating it would likely involve updating https://github.com/OSGeo/OSGeoLive/blob/master/bin/install_osm.sh 

@althio, @darkblue-b, thoughts?

**Comment by cvvergara** [comment]
FYI
The "large" images used in josm_quickstart are around 90K to 120K, the sample.osm file is just 16.4 K
That file is smaller than an image.

**Comment by kalxas** [comment]
Big +1 to use the small sample that LearnOSM requires.
I will issue a pull request to include it.

**Comment by althio** [comment]
@camerons 
Yes I think it is a good idea to include the `sample.osm` dataset.
It is a very small (as noted by @cvvergara) "synthetic" dataset, to discover main tools without the clutter of a real mapped city - which can be overwhelming.

@kalxas 
if you include the file, please modify the link 
at josm_quickstart.rst @ L.53
> -  Download the file here: sample.osm <path/to/sample.osm>

**Comment by kalxas** [comment]
https://github.com/OSGeo/OSGeoLive/commit/17e606d6e2b1a12d69536e5d2fceb96bf78c1ab5

**Pull request merged**